lib/src/test/java/uk/gov/di/ipv/core/library/helpers/RequestHelperTest.java

```diff
@@ -210,4 +212,24 @@ void getFeatureSetShouldReturnDefault() {
 
         assertEquals("default", RequestHelper.getFeatureSet(event));
     }
+
+    @Test
+    void getPathParametersShouldReturnPathParameters() {
+        var event = new APIGatewayProxyRequestEvent();
+        event.setPathParameters(Map.of("criId", DCMAW_CRI));
```

Comment:
See comment in JourneyRequest in which case you would test by setting the `journey` field to `/journey/cri/build-oauth-request/dcmaw`

Reply:
This test was added for the 'criId' path variable